examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
Agreed, `show`, `save`, `push` etc. should all be the cross API ways to show, save, and push objects and plots regardless of how they were created.